spring-rabbit/src/main/java/org/springframework/amqp/rabbit/annotation/RabbitListener.java

```diff
@@ -332,4 +332,18 @@
 	 */
 	String converterWinsContentType() default "true";
 
+	/**
+	 * Override the container factory's {@code batchListener} property. The listener
+	 * method signature should receive a {@code List<?>}; refer to the reference
```

Comment:
Just for my info: do we validate somewhere that listener method has a `List` signature for a batch mode?

Reply:
We currently don't; I'll take a look to see if it's feasible.

Reply:
No problem! Perhaps other issue/PR.